test_programs/execution_report.sh

```diff
@@ -44,8 +44,27 @@ for dir in ${tests_to_profile[@]}; do
       exit 1
     fi
 
-    COMPILE_TIME=$((time nargo execute --silence-warnings) 2>&1 | grep real | grep -oE '[0-9]+m[0-9]+.[0-9]+s')
-    echo -e " {\n    \"artifact_name\":\"$PACKAGE_NAME\",\n    \"time\":\"$COMPILE_TIME\"" >> $current_dir/execution_report.json
+
+    NUM_RUNS=1
```

Comment:
This report could be combined with `compilation_report.sh`. I would make an issue to combine them, but we already have this issue to use our tracing logs for generating these reports (https://github.com/noir-lang/noir/issues/6844), which should also combine them.